### PR TITLE
Gaussian blur radius fix

### LINKED ISSUE #2239: Gaussian Blur inconsistency with PIL
I've noticed that performing a gaussian blur with `pygame.transform.gaussian_blur` makes a less blurry image than PIL's gaussian blur when the same radius is used. If the radius of the pygame blur is doubled, then they match up correctly. 

I assume this isn't intended behaviour? Is it possible that the radius was not doubled by Pygame CE when it should have been, or something like that?

Blurring a pygame surface with PIL:

```py
img = Image.frombytes(
    "RGBA", surface.get_size(), pygame.image.tostring(surface "RGBA", False)
)
img = img.filter(ImageFilter.GaussianBlur(radius=10))
blurred_surface = pygame.image.frombuffer(
    img.tobytes(), img.size, "RGBA"
)
```

With Pygame CE:

```py
blurred_surface = pygame.transform.gaussian_blur(surface, 10)
```

Side-by-side comparison, with both radii set to 10:

Both radii set to 10:

PIL with a radius of 10, and Pygame with a radius of 20:

### PR DESCRIPTION
**Pull request by yunline**
Fixes #2239
image
```py
import pygame
import sys

pygame.font.init()

pygame.display.init()
dsf=pygame.display.set_mode((1280,720))
sf=pygame.image.load("lena.png")

radius = 10

from PIL import Image,ImageFilter
img = Image.frombytes(
    "RGBA", sf.get_size(), pygame.image.tobytes(sf, "RGBA", False)
)
img = img.filter(ImageFilter.GaussianBlur(radius=radius))
sf_blur1 = pygame.image.frombuffer(
    img.tobytes(), img.size, "RGBA"
)

sf_blur2 = pygame.transform.gaussian_blur(sf,radius)

dsf.fill((0,150,0))
dsf.blit(sf_blur2,(0,0,1,1))
dsf.blit(sf_blur1,(640,0,1,1))
pygame.display.update()

while 1:
    for event in pygame.event.get():
        if event.type==pygame.QUIT:
            sys.exit()
```

### CODE REVIEW

**Comment by Starbuck5** [comment]
This changes the behavior of an existing function. I guess it's okay in this case because it's a very new function? Just introduced in 2.2.0? That was still 2.5 months ago...

Nevertheless, this should definitely get a notice in the docs, documenting the change between versions.

File: src_c/transform.c [hunk 1]
```diff
@@ -3069,21 +3069,20 @@ gaussian_blur(SDL_Surface *src, SDL_Surface *dst, int radius, SDL_bool repeat)
     int dst_pitch = dst->pitch;
     int src_pitch = src->pitch;
     int i, j, x, y, color;
+    int kernel_radius = sigma * 2;
```
[the remark this answers is not in the document]
Comment (yunline) [reply src_c/transform.c:3072]: Nope, it's kernel_radius. The real kernel size is `kernel_radius*2+1`

**Review by Starbuck5 — APPROVED** [review]
Ok, looks good.

**Review by ankith26 — APPROVED** [review]
thanks! :partying_face:

**Review by MyreMylar — APPROVED** [review]
Yep, this fix works for me and gaussian blur now seems to be consistent with other implementations.

**Pull request merged**